<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDatabaseWindow</class>
 <widget class="QDialog" name="DeleteDatabaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuń bazę danych</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyjście</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>261</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>image_label</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>exit_button</zorder>
  <zorder>save_button</zorder>
  <zorder>listView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
